frontend/src/store/index.ts

```diff
@@ -1,26 +1,27 @@
+import { createContext } from 'react';
 import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/Registry';
 import promiseMiddleware from 'redux-promise-middleware';
 import { notificationsMiddleware } from '@redhat-cloud-services/frontend-components-notifications/notificationsMiddleware';
 import { SDKReducers } from '@openshift/dynamic-plugin-sdk-utils';
 import thunk from 'redux-thunk';
 
+export type Registry = {
+  getRegistry: () => {
+    getStore: () => unknown;
```

Comment:
Not blocking, but it would be nice to clean up these types

Reply:
Types has been updated!